packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/template.success.js

```diff
@@ -38,8 +38,23 @@ const isPending = (t) => t.state === 'processing'
 const isCompleted = (t) => t.state !== 'processing'
 
 const Success = props => {
-  const { changeBuySellTabStatus, fetchQuote, fetchSellQuote, refreshQuote, submitQuote, submitSellQuote, handleTrade, quoteR, sellQuoteR, base, errors, showModal, handleTradeDetailsClick, ...rest } = props
-
+  const {
+    changeBuySellTabStatus,
+    payment,
+    fetchQuote,
+    fetchSellQuote,
+    refreshQuote,
+    submitQuote,
+    submitSellQuote,
+    handleTrade,
+    quoteR,
+    sellQuoteR,
+    base,
+    errors,
+    showModal,
+    handleTradeDetailsClick,
+    ...rest } = props
+  console.log('sfox content success:', payment)
```

Comment:
Small thing but think we should prepend 'buy' to fetchQuote, refreshQuote, quoteR. Makes it more readable

Reply:
👍 agree